doc/design/OrionHDFSSink.md

```diff
@@ -225,13 +232,13 @@ Use `OrionHDFSSink` if you are looking for a JSON or CSV-based document storage
 ###<a name="section4.1"></a>`OrionHDFSSink` class
 As any other NGSI-like sink, `OrionHDFSSink` extends the base `OrionSink`. The methods that are extended are:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/b13eedc0daf7fb440f3d3decb43b596a54cb9f1f